tools/operator/networkspec/structs.go

```diff
@@ -178,3 +180,125 @@ type ConnectionProfile struct {
 	CA            map[string]CertificateAuthority `yaml:"certificateAuthorities"`
 	Organizations map[string]Organization         `yaml:"organizations"`
 }
+
+//ChaincodeID --
+type ChaincodeID struct {
+	Id      string `yaml:"id,omitempty"`
+	Version string `yaml:"version,omitempty"`
+}
+
+// Caliper related struct
+//Orderer --
+type CaliperOrderer struct {
+	URL         string `yaml:"url"`
+	GrpcOptions struct {
+		SslTarget string `yaml:"ssl-target-name-override"`
+	} `yaml:"grpcOptions"`
+	TLSCACerts struct {
+		Pem string `yaml:"pem"`
+	} `yaml:"tlsCACerts"`
+}
+
```

Comment:
This isn't you, I know you were just copying whats already done, but we need to come back at a later date and remove the embedded structs from all of these structs. It makes constructing objects very difficult to do. No need to correct it now, as its a much bigger problem introduced earlier on, just making it known that this isn't great.

Reply:
we can come back to this later